src/exporter.js

```diff
@@ -133,9 +136,22 @@ export default class {
   _tagsToStr(tags=[]){
     return tags.map(tag => tag.replace(/ /g, '_')).join(' ');
   }
-}
 
-export const rand = () => Math.random() * 100000000 | 0;
+  _getId(table, col, ts) {
+    const query = `SELECT ${col} from ${table} WHERE ${col} >= ${ts}`;
```

Comment:
There is also `ORDER BY` statement in my query which allows selecting the greatest value from columns. (The greatest value + 1) === non-existing ID. 

This is possible because `sql.js` is synchronous for some reason.

---

I wonder would the prepared statements also work for other generated parts of query e.g. `SELECT :col from :table WHERE :col`?